templates/taxi/index.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <h1>Welcome to the Taxi Service Dashboard</h1>
+    <ul>
+        <li>Number of cars: {{ num_cars }}</li>
+        <li>Number of drivers: {{ num_drivers }}</li>
+        <li>Number of manufacturers: {{ num_manufacturers }}</li>
+    </ul>
+{% endblock %}
```

Comment:
The `content` block is overridden correctly to display the number of cars, drivers, and manufacturers. This fulfills the requirement to show this information as a list on the home page.